packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -101,10 +101,22 @@ export type Endpoints = {
     "GET",
     BlockArgs,
     {params: {block_id: string}},
-    BeaconBlockBody["attestations"],
+    AttestationListPhase0,
```

Comment:
while this is correct I am not sure it's good to decouple the type here from the block body type

Could use the new forks as generic typing to pin down the forks supported by the api
```suggestion
    BeaconBlockBody<ForkPre....whatever we come up with>["attestations"],,
```

I noticed we don't have a Fork pre type for electra yet though, see https://github.com/ChainSafe/lodestar/blob/nc/attestation-api-v2/packages/params/src/forkName.ts